jme3-core/src/main/java/com/jme3/scene/shape/Surface.java

```diff
@@ -45,10 +45,7 @@
 import com.jme3.util.BufferUtils;
 import java.io.IOException;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
JME's preferred code style is based on [Google style](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) which specifies "No wildcard imports."

Reply:
Fixed from where I was responsible (3 classes)

Reply:
Thank you!